mrp_progress_button/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "author": "Akretion, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/manufacture",
     "category": "Manufacturing",
-    "version": "14.0.1.0.0",
+    "version": "14.0.1.1.0",
```

Comment:
A small comment, no need to change it this time. But the version bumping is done with the ocabot merge command.

In this case we will use `ocabot merge nobump` because you already bumped the version, but that is a task that is better to leave for the bot (`ocabot merge minor` would have done the job).